woocommerce/woocommerce-admin
# Update Storybook to latest

Fixes #5103 

Storybook has a new major version out, and the APIs & recommended addons are evolving, so I think it's a good idea to proactively keep up to date. This PR updates Storybook to latest, but doesn't adopt any new features or change existing stories - that can come later.

### Screenshots

<img width="937" alt="Screen Shot 2020-09-08 at 1 56 41 PM" src="https://user-images.githubusercontent.com/4167300/92425373-22ab3980-f1db-11ea-9116-a886d4d3a6a9.png">

### Detailed test instructions:

- Get on this branch.
- Update dependencies, e.g. `npm install`.
- Build & run storybook: `npm run storybook`.
- Browse storybook and confirm everything's working correctly.

Bonus points: try adding a new story or adding more examples to an existing story.

### Changelog Note:

n/a (no merchant facing change/impact) ?
